### ISSUE TITLE
Discussion: Considerations for Accessibility in Hubs

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#1899]
Following up on the [W3C workshop on Inclusive Design for Immersive Technologies](https://www.w3.org/2019/08/inclusive-xr-workshop/), I wanted to kick off a conversation about accessibility in Hubs and areas that we can improve the platform so that it is more widely available. This will be an ongoing initiative and cover a lot of areas, but to start with, I wanted to have a central place to discuss, at a high level, the different things we should be researching and understanding around accessible content on the Hubs platform.

I've shared my initial thinking [on the Hubs Wiki here](https://github.com/mozilla/hubs/wiki/Considerations-for-Accessibility-in-Hubs) but wanted to ensure that this was also captured in a format that would allow for discussion. 

There is now an 'accessibility' tag for issues in GitHub in this repository.

**Comment by misslivirose** [comment]
Related: #1883

**Comment by robertlong** [comment]
This is great! I'd also like to open up discussion around accessibility focused on people with reduced mobility and general ergonomics. I think there are things we can do within our UX design to make things more accessible in VR, in 3DOF and 6DOF headsets.

**Comment by misslivirose** [comment]
Definitely agree - I think that maybe I'll make #1883 more general purpose for mobility accessibility, since it is just one possible implementation for approaching that
